pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -3020,11 +3020,10 @@ typedef struct pjsua_transport_config
     /**
      * Specify the port range for socket binding, relative to the start
      * port number specified in \a port. Note that this setting is only
-     * applicable when the start port number is non zero.
+     * applicable to media transport when the start port number is non zero.
+     * Refer to pjsua_acc_config.rtp_cfg documentation for the sample
+     * implementation.
```

Comment:
What if `Refer to ...` is replaced by something like `Media transport is configurable via account setting, i.e: pjsua_acc_config.rtp_cfg, please check the media transport config docs for more info`.